City-of-Helsinki/drupal-helfi-platform-config
# Uhf 149 elasticsearch

To test:

- Clone or setup HELfi platform with oneliner
- switch branches: `git checkout UHF-149_Elasticsearch && composer require drupal/helfi_platform_config:dev-UHF-149_Elasticsearch`
- run `make new`
- enable migrate_tools and helfi_search (will also enable helfi_ahjo among other dependencies)
- check that the Elasticsearch server is up: _admin/config/search/elasticsearch-connector_, Cluster status should be green/yellow. If not, check docker logs for error messages
- do a test migration for Ahjo issues first (`drush migrate:import --group ahjo --limit 10`)
- check _/admin/config/search/search-api/index/pages_, you should see 2 separate lines for different content types, with index count something else than zero
- check _/search-index-items_, table should be populated with ahjo issues and content nodes
